### PR TITLE
122 marketplace routes specification

### LINKED ISSUE #122: Marketplace Routes Specification

### PR DESCRIPTION
**Pull request by VanJackal**
should wait for #152 to be merged

closes #122

### CODE REVIEW

File: critterSpec.yaml [hunk 1]
```diff
@@ -1,936 +1,1132 @@
...
-                $ref: '#/components/schemas/Card'
-      operationId: get-cards-cardid
-      x-stoplight:
-        id: hnsd41zsb96gl
-  '/users/{userid}/cards':
+          description: Created
+      summary: Create new pack
+      tags:
+        - packs
+  /packs/{packid}:
```
Comment (kivine99) [inline critterSpec.yaml:522]: I think this is missing the request body

File: critterSpec.yaml [hunk 2]
```diff
@@ -1,936 +1,1132 @@
...
+        - amount
+    VendorRep:
+      type: object
+      properties:
+        level:
+          type: integer
+        next:
+          type: integer
+        prev:
+          type: integer
```
Comment (kivine99) [inline critterSpec.yaml:326]: I'm not exactly sure what these are--a more descriptive name could be helpful.

File: critterSpec.yaml [hunk 3]
```diff
@@ -1,936 +1,1132 @@
...
+        required: true
+        description: ""
         content:
           application/json:
             schema:
-              type: object
-              $ref: '#/components/schemas/RegisterDetails'
-  /users/auth/logout:
+              $ref: "#/components/schemas/Offer"
+  /vendors/{vendorid}/offers/{offerid}:
```
Comment (kivine99) [inline critterSpec.yaml:1017]: Will the API know which user is purchasing?

Comment (VanJackal) [reply critterSpec.yaml:1017]: yes, its based on who is signed in

**Pull request merged**